articles/azure-sql/virtual-machines/windows/availability-group-load-balancer-portal-configure.md

```diff
@@ -68,7 +68,7 @@ First, create the load balancer.
    | --- | --- |
    | **Name** |A text name representing the load balancer. For example, **sqlLB**. |
    | **Type** |**Internal**: Most implementations use an internal load balancer, which allows applications within the same virtual network to connect to the availability group.  </br> **External**: Allows applications to connect to the availability group through a public Internet connection. |
-   | **SKU** |**Standard**: Required if your SQL instances are in a different availability set than the load balancer. </br> **Basic**: Default option. |
+   | **SKU** |**Standard**: Required if your SQL instances are in a different availability zone than the load balancer. </br> **Basic**: Default option. |
```

Comment:
| **SKU** |**Basic**:  Default option. Only valid if SQL Server instances are in the same availability set.</br> **Standard**: Preferred. Valid if SQL Server instances are in the same availability set. Required if your SQL Server instances are in different availability zones.